extensions/amp-fx-collection/amp-fx-collection.md

```diff
@@ -96,7 +96,7 @@ This parameter lets you vary the animation's speed over the course of its durati
 * “ease-out” - cubic-bezier(0.40, 0.00, 0.40, 1.00)
 or specify a `custom-bezier()` input
 
-##### data-margin
+##### data-margin-start
 
 This parameter determines when to trigger the timed animation. The value specified in `<percent>` dictates that the animation should be triggered when the specified amount of the element being targetted is visible. The default value is `5%`
```

Comment:
Nit: 

This doesn't have a lot to do with your rename, but I realized that the naming is confusing because I don't really understand what this variable does, specifically what it has to do with margins. Would you mind making this explicit by adding a code example and describing what that code example does? I didn't really understand what `<percent>` was referring to without an example. 

```
<div amp-fx="fade-in" data-margin-start="5%">
    <amp-img width="1600" height="900" layout="responsive" src="https://picsum.photos/1600/900?image=1069"></amp-img>
  </div>
```
> In this example, the animation will be triggered when 5% of the element's area is visible. 

Also, it seems like not all of these parameters are mandatory. Could you add "(optional)" behind all of the optional parameters?

Reply:
Made both the changes you asked for:
* Added an example
* Specified that the attributes aren't mandatory.